.github/workflows/gradle-run-unit-tests.yml

```diff
@@ -8,6 +8,10 @@ on:
         - 'develop'
   workflow_call:
 
+concurrency:
+    group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+    cancel-in-progress: true
```

Comment:
What's this for? to cancel an ongoing GH action if a new commit arrives?

Reply:
oh yeah, the same filter we have on kalium, to save some 💸